ktxspec.adoc

```diff
@@ -977,6 +977,9 @@ _dfDescriptorBlock_.
     - If <<_vkformat,`vkFormat`>> is not one of the `+*_HLG{,_*}+`
       formats and an HLG variant of that format exists,
       `transferFunction` should not be `KHR_DF_TRANSFER_HLG`.
+* If <<_vkformat,`vkFormat`>> is one of the `+*_[SU]INT{,*}+` formats
+  or one of the depth/stencil formats `transferFunction` must be
```

Comment:
To avoid any ambiguities, consider expanding to:
> depth, stencil, or combined depth/stencil

Reply:
Thanks for the suggestion.